platform/Tcp.roc

```diff
@@ -205,12 +204,12 @@ connect_err_to_str = \err ->
 ## ```
```

Comment:
docs on line 185-187 have space application of tags instead of PNC

Reply:
Fixed in 77d5ee0